controller/service.go

```diff
@@ -32,8 +32,9 @@ import (
 )
 
 const (
-	annotationAddressPool     = "metallb.universe.tf/address-pool"
-	annotationLoadBalancerIPs = "metallb.universe.tf/loadBalancerIPs"
+	annotationAddressPool        = "metallb.universe.tf/address-pool"
+	annotationLoadBalancerIPs    = "metallb.universe.tf/loadBalancerIPs"
+	annotationIPAllocateFromPool = "metallb.universe.tf/allocate-pool-from"
```

Comment:
Can we call this "ip-allocated-from-pool"?

Reply:
Sure!